File: frontend/src/models/SaleModel.js
import axios from 'axios';

const API_URL = 'http://localhost:8080/api/sales';

export const sellArtwork = (artworkId, employeeId, saleDate, price) => {
    return axios.post(API_URL, {
        artworkId,
        employeeId,
        saleDate,
        price
    });
};

File: frontend/GalleryFrontend/Models/SaleModel.cs
namespace GalleryFrontend.Models
{
    public class SaleModel
    {
        public int Id { get; set; }                 
        public int ArtworkId { get; set; }
        public int EmployeeId { get; set; }
        public DateTime SaleDate { get; set; }
        public double Price { get; set; }

        public string ArtworkTitle { get; set; } = "";
        public string EmployeeName { get; set; } = "";
    }
}